DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/FakeTopLevelConverter.cs

```diff
@@ -6,11 +6,11 @@
 namespace Speckle.Converters.ArcGIS3;
 
 [NameAndRankValue(nameof(String), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
-public class FakeConversion : IHostObjectToSpeckleConversion, IRawConversion<String, Point>
+public class FakeTopLevelConverter : IToSpeckleTopLevelConverter, ITypedConverter<String, Point>
```

Comment:
This kinda gives me a headache 🤕 Not your doing, just flagging this to myself to open a ticket about it.